### DISCUSSION TITLE
Filter connection requests between users

### DISCUSSION THREAD

**Discussion by lukadriel7** [peers/peerjs#1306] [Q&A]
Hello,
I was wondering if there was a way to filter which user can connect to another. A system like users cannot message other users who blocked them. I would like to keep the implementation on the server side. I am hosting my own signaling server integrated with NestJS. Ideally, when a user makes a connection request to another, I want to check if the user is blocked and if that's the case prevent the connection. Did anyone implement something similar ?

**Comment by lherman-cs** [comment]
@lukadriel7 I've implemented something similar to block other users from reaching each other based on the backend policy. I'm happy to discuss this more over Discord. This is my Discord username, lherman_cs
